### ISSUE TITLE
AttributeError: 'module' object has no attribute 'NullHandler'

### ISSUE THREAD

**Issue by DavidAwad** [geduldig/TwitterAPI#54]
Sorry, just never had the chance to comment / post the version. 2013 macbook air, newest osx. Let me know if you need anything else. Thanks! 

```python
python --version
Python 2.7.9
python server.py
Traceback (most recent call last):
  File "server.py", line 8, in <module>
    from TwitterAPI import TwitterAPI
  File "/Library/Python/2.7/site-packages/TwitterAPI/__init__.py", line 12, in <module>
    logging.getLogger(__name__).addHandler(logging.NullHandler())
AttributeError: 'module' object has no attribute 'NullHandler'
```

**Comment by geduldig** [comment]
I need a short code sample that produces this error.

**Comment by DavidAwad** [comment]
It's just importing the api. for some reason that causes the error. It's possible it's being caused by a buggy python installation? I'm not sure as I'm not getting any other problems with other packages.

**Comment by geduldig** [comment]
As I said in Issue #53, I tried and could not reproduce your error. You probably are doing more than "just importing the api" if your traceback points to line 8 of your code. You can either send me a short sample, or try running a python script with just the one line of code `from TwitterAPI import TwitterAP` and see if you still get an error.

**Comment by geduldig** [comment]
One more thing. I have an older mac that came with an older version of Python 2.7. When I installed Python 2.7.9 my path still pointed to the older version. I had to modify my PATH environment variable to point to the new version. I don't think you are having this issue since you said when you ran `python --version` it outputted 2.7.9. But, I may be just confusing the issue since all 2.7 versions of python include `NullHandler`.
